src/App.php

```diff
@@ -1023,10 +1040,8 @@ public function is_friendica_app()
 	 * This isn't a perfect solution. But we need this check very early.
 	 * So we cannot wait until the modules are loaded.
 	 *
-	 * @return bool Is it a known backend?
 	 */
-	public function is_backend()
-	{
+	private function checkBackend($backend) {
```

Comment:
The parameter `$backend` isn't used in this method and not documented.

Reply:
it is used at `return`.
I added a doc for it